WordPressLoginFlow/src/main/java/org/wordpress/android/login/LoginListener.java

```diff
@@ -61,7 +61,11 @@ void needs2faSocial(String email, String userId, String nonceAuthenticator, Stri
     // Login Site Address input callbacks
     void alreadyLoggedInWpcom(ArrayList<Integer> oldSitesIds);
     void gotWpcomSiteInfo(String siteAddress);
-    void gotConnectedSiteInfo(@NonNull String siteAddress, @Nullable String redirectUrl, boolean hasJetpack);
+    default void gotConnectSiteInfo(@NonNull ConnectSiteInfoResult result) {
+        gotConnectedSiteInfo(result.getUrl(), result.getUrlAfterRedirects(), result.getHasJetpack());
+    }
+
```

Comment:
With these default implementations, we can avoid making this a breaking change for the client apps, for now, the apps will keep overriding the old version `gotConnectedSiteInfo`, while for WCAndroid, we will soon migrate to the new one `gotConnectSiteInfo` that passes the new model `ConnectSiteInfoResult` which would allow us to know if the site is suspended.